catalogd/cmd/catalogd/main.go

```diff
@@ -147,7 +153,7 @@ func newVersionCmd() *cobra.Command {
 	}
 }
 
-func run(cfg *config) error {
+func run(cfg *config, ctx context.Context) error {
```

Comment:
By convention `context.Context` should always be the first argument in the function signature.